test.sh

```diff
@@ -145,7 +145,7 @@ while getopts lueciosvgrnhp:f:-: OPT; do
     s | start-py )                   RUN+=("start") ;;
     v | gomod-vendor )               RUN+=("gomod-vendor") ;;
     g | generate )                   RUN+=("generate") ;;
-    r | rpm )                        RUN+=("rpm") ;;
+    m | make-artifacts )             RUN+=("make-artifacts") ;;
```

Comment:
You need to replace `r` with `m` on line `131` or the short-opt will never be parsed.